### ISSUE TITLE
Too many beacon-related messages

### ISSUE THREAD

**Issue by MarcRiera** [leezer3/OpenBVE#64]
The recent addition of the ability to display messages from train plugins offers a lot of potential. However, since that was implemented, I have been getting messages related to beacons on many routes, no matter which plugin is used at the time.

![captura de pantalla de 2016-04-12 21-38-52](https://cloud.githubusercontent.com/assets/12414284/14473374/f97ec56e-00f7-11e6-9e8c-c016f92adb09.png)

I am not sure whether this was intended or not, but I suppose probably not, since some routes make an extensive usage of beacons and the screen becomes quickly cluttered with messages.

Maybe this was supposed to get displayed in the debug (F10) screen and somehow managed to appear in the main screen?

**Comment by leezer3** [comment]
Stupid question- Release or debug build?

If they only display in debug builds, I may have inadvertently fixed a
'feature' :)

I know I did a bunch of fiddling with messages, as I moved the color table
and some other stuff into the API so plugins could access it easily.

Will take a closer look in the debugger either tonight or tomorrow.
On 12 Apr 2016 20:49, "Marc Riera" <notifications@github.com> wrote:

The recent addition of the ability to display messages from train plugins
offers a lot of potential. However, since that was implemented, I have been
getting messages related to beacons on many routes, no matter which plugin
is used at the time.

[image: captura de pantalla de 2016-04-12 21-38-52]
<https://cloud.githubusercontent.com/assets/12414284/14473374/f97ec56e-00f7-11e6-9e8c-c016f92adb09.png>

I am not sure whether this was intended or not, but I suppose probably not,
since some routes make an extensive usage of beacons and the screen becomes
quickly cluttered with messages.

Maybe this was supposed to get displayed in the debug (F10) screen and
somehow managed to appear in the main screen?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub
<https://github.com/leezer3/OpenBVE/issues/64>
